src/PrestaShopBundle/Form/Exception/DataProviderException.php

```diff
@@ -31,6 +31,7 @@
 use PrestaShop\PrestaShop\Core\Domain\Exception\DomainException;
 use Throwable;
 
+/** @deprecated and will be removed in 9.0 */
```

Comment:
You can call @trigger_error before the class to make sure in will appear in the logs.